### ISSUE TITLE
Content Tweaks

### ISSUE THREAD

**Issue by kaniasty** [codeforboston/plogalong#262]
On the plog screen:

In plogging items, replace Needles with Metal/Junk
- also remove the Danger Pay achievement. 

Remove the horizontal gray line between What were you up to and icons, as well as between Who helped and icons. 

Change 'Running' to 'Jogging'
Change 'Swimming' to 'On the beach' 
Change 'Canoeing' to 'Paddling'

Change My dog to 'My dog(s) cheered me on.'
Change A child to 'Kid plogger(s).'
Change A friend to 'Friend or partner.'
Change A team to 'It was a group effort.' 

Switch the font sizes between 'What did you clean up?' etc. 
and Walking, etc.
Basically, the header should be larger than the helper text below the icons. However, keep the headers bold, and the helper text unbold.
